### ISSUE TITLE
Breadcrumbs:[A11y Issue] Visually apparent "list" of links not made programmatic list in breadcrumb trail

### ISSUE THREAD

**Issue by sleric1024** [SAP/ui5-webcomponents-react#1474]
**Describe the bug**
The breadcrumb trail is not contained within a navigation landmark, missing an accessible name and the visually apparent "list" of links hasn't been made a "list" programmatically.

**Isolated Example**
https://05sbk.csb.app/

**To Reproduce**
1. Go to https://05sbk.csb.app/ with screen readers
2. Focus on the first navigation link

**Expected behavior**
1) The breadcrumb trail has to be within a <nav> landmark or must have role="navigation" with an aria-label="breadcrumb trail"
2) The "list" of links within the trail should be an unordered list of links
3) The current page text within the trail should have aria-current="page"
4) Could using arrow keys to change the focus on Breadcrumbs
Please refer to the UI5 reference below and match the implementation to align with it
https://ui5.sap.com/#/entity/sap.m.Breadcrumbs/sample/sap.m.sample.Breadcrumbs

**Screenshots**
UI5 Control Behavior
![image](https://user-images.githubusercontent.com/5724592/113103257-a1c49580-9231-11eb-8606-9da51e011ee2.png)

- Step1: Screen readers text for the first focus on Breadcrumbs
- Step2: Focusing changing
- Step3: The focusing on current page. 

**UI5 Web Components for React Information**
`@ui5/webcomponents` version: lasted version
`@ui5/webcomponents-react` version: lasted version
Operating System: Window 10
Browser: Chome

**Additional context**
I am from SuccessFactors.

**Comment by Lukas742** [comment]
released in [0.15.3](https://github.com/SAP/ui5-webcomponents-react/releases/tag/v0.15.3)
